view/src/components/produce.js

```diff
@@ -492,79 +513,97 @@ class produce extends Component {
-                                                </Box>
-                                                <Box p={3}>
-                                                    <Typography className={classes.pos} color="textSecondary">
-                                                        Internal Statistics:
-                                                    </Typography>
-                                                    <Typography variant="body2" component="p">
-                                                        Amount Moved by FarmLink (lbs): {produce.amountMoved}
-                                                    </Typography>
-                                                </Box>
-                                            </Box> 
-                                            
-                                        </CardContent>
-                                        <CardActions>
-                                            <Button size="small" color="primary" onClick={() => this.handleViewOpen({ produce })}>
-                                                {' '}
-                                                View{' '}
-                                            </Button>
-                                            <Button size="small" color="primary" onClick={() => this.handleEditClickOpen({ produce })}>
-                                                Edit
-                                            </Button>
-                                            <Button size="small" color="primary" onClick={() => this.deleteTodoHandler({ produce })}>
-                                                Delete
-                                            </Button>
-                                        </CardActions>
-                                    </Card>
-                                </Grid>
-                            ))}
-                        </Grid>
+								<SearchResults
+									value={value}
+									data={data}
+									renderResults={results => (
+										<div>
+                            				{results.map((produce) => (
+												<Grid item xs={12}key={produce.produceId}>
+													<Card className={classes.root} variant="outlined">
+														<CardContent>
+															<Typography variant="h5" component="h2">
+																{produce.name}
+															</Typography>
+															<Box display="flex" flexDirection="row" flexWrap="wrap" p={0} m={0}>
+																<Box p={3}>
+																	<Typography className={classes.pos} color="textSecondary">
+																		Shipping Temperatures in Reefer (°F): 
+																	</Typography>
+																	<Typography variant="body2" component="p">
+																		Maintenance Temperature: {produce.shippingMaintenanceTemperatureLow} - {produce.shippingMaintenanceTemperatureHigh}
+																		<br />
+																		Preset Temperature: {produce.shippingPresetTemperature}
+																	</Typography>
```

Comment:
You might end the Typography element before the <br> and start a new one. It feels weird to include a <br> in with the Typography element.

Reply:
LOL my `<br>`s became actual line breaks...

Reply:
(@andrewhojel if you wanna make this change on your branch and then I'll edit it once the merge conflict shows up)

Reply:
I will look into this!